app/utils/nav-builder.js

```diff
@@ -49,7 +49,7 @@ const serviceNavigationItems = (currentPath, permissions, type) => {
   navigationItems.push({
     id: 'navigation-menu-settings',
     name: 'Settings',
-    url: type === 'card' ? paths.settings.index : paths.apiKeys.index,
+    url: paths.settings.index,
```

Comment:
Removed the DD code here - now only handles the `card` payments.

Reply:
It won't allow me to comment on it, but need to update line 56: `paths.apiKeys` -> `paths.account.apiKeys` to make sure the "Settings" tab is highlighted on the top nav bar when you're on this page.